snippets/core/system-text-json/csharp/DateTimeOffsetConverter.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Globalization;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace SystemTextJsonSamples
+{
+    public class DateTimeOffsetConverter : JsonConverter<DateTimeOffset>
+    {
+        public override DateTimeOffset Read(
+            ref Utf8JsonReader reader,
+            Type typeToConvert,
+            JsonSerializerOptions options)
+        {
+            return DateTimeOffset.ParseExact(reader.GetString(),
```

Comment:
This might be a good candidate for an expression bodied member:

```csharp
public override DateTimeOffset Read(
    ref Utf8JsonReader reader,
    Type typeToConvert,
    JsonSerializerOptions options) => 
        DateTimeOffset.ParseExact(reader.GetString(),
        "MM/dd/yyyy", CultureInfo.InvariantCulture);
```

But I'm not sure, because the extra line breaks make it not an obvious choice.

Reply:
It eliminates two nearly empty lines of code, so it seems a reasonable choice.